rebar.config

```diff
@@ -1,5 +1,5 @@
 {plugin_dir               , "src"}.
-{erl_opts                 , [debug_info]}.
+{erl_opts                 , [debug_info, {platform_define, "^(19|2)", rand_only}]}.
```

Comment:
This will break once Erlang hits 30, there should be a better regex.

Reply:
How about something like https://github.com/apache/couchdb-mochiweb/blob/999f464ec79a9a54f59bfeff21728d0bba6e857a/rebar.config#L5 ?

Reply:
I like! Single out the ones that doesn't have rand instead of the ones that have it!